internal/services/redis/redis_cache_data_source.go

```diff
@@ -71,6 +72,11 @@ func dataSourceRedisCache() *pluginsdk.Resource {
 				Computed: true,
 			},
 
+			"non_ssl_port_enabled": {
```

Comment:
can we wrap the schema in the beta flag here to make it easier to find and remove later?

Reply:
Thanks @jackofallops - I spotted another TODO that I had missed, if you wouldn't mind having another quick look!